libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -430,7 +430,11 @@ export class SkyDateRangePickerComponent
         this.#changeDetector.markForCheck();
       }
 
-      this.hasErrors = !!control.errors && (control.touched || control.dirty);
+      if (this.#startDateControl.dirty || this.#endDateControl.dirty) {
+        this.#calculatorIdControl.markAsDirty();
+      }
```

Comment:
if either of the two date input boxes in date range picker are dirty, make sure the first input box (the dropdown) is also marked as dirty. That first dropdown is where all errors are shown for now.